modules/github-bots/main.tf

```diff
@@ -1,4 +1,5 @@
 resource "google_service_account" "sa" {
+  count        = var.service_account_email == "" ? 1 : 0
```

Comment:
as this changed the resource path, 
can you add a moved block
https://developer.hashicorp.com/terraform/language/modules/develop/refactoring#enabling-count-or-for_each-for-a-resource

```
moved {
  from = google_service_account.sa
  to = google_service_account.sa[0]
}
```

Reply:
Oh! nice!